TopMovies/Connector/MovieVCConnector.swift

```diff
@@ -7,7 +7,9 @@
 
 class MovieVCConnector<Provider: StoreProviderProtocol>: BaseConnector<MovieVCProps, Provider>
 where Provider.ExpectedStateType == MainState {
+  @Inject private var router: RouterProtocol
   private let movieId: MoviePreview.ID
+  private var isMovieSent = false
```

Comment:
What `isMovieSent` for ?

Reply:
> What `isMovieSent` for ?

When the user reopens some movie detail, I have a situation where the Movie exists in relational, so the props are sent, but then the state is updated according to the RequestAction, and I get it again, the Movie exists again, so the props are created and sent again, but it's the same props, so I do the same job twice. That's why I have a flag there